src/MLJInterface.jl

```diff
@@ -189,7 +189,7 @@ function update(mlj_model::MLJInterface.MODELS, verbosity::Int, fitresult, cache
         report = LightGBM.train!(old_lgbm_model, additional_iterations, String[], verbosity, LightGBM.Dates.now())
         # should probably dump report too, its just an empty dict...
         fitresult = (old_lgbm_model, old_classes, mlj_model)
-        return (fitresult, cache, report)
+        return (fitresult, cache, (report,))
     end
```

Comment:
If there is no report, you can just return `NamedTuple()`. Otherwise it should be a named-tuple with informative keys.

Reply:
So I think there is in theory a report if user specifies the metrics that underlying library evaluates but I never fully checked it, and I'd rather not mess around with it now